modules/persistent-storage-csi-google-cloud-file-delete-instances.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-google-cloud-file.adoc
+
+:_content-type: PROCEDURE
+[id="persistent-storage-csi-gcp-cloud-file-delete-instances_{context}"]
+= Deleting GCP Filestore instances
+
+Typically, if you destroy a cluster, the {product-title} installer deletes all of the cloud resources that belong to that cluster. However, when a cluster is destroyed, Google Compute Platform (GCP) Filestore instances are not automatically deleted, so you must manually delete all persistent volume claims (PVCs) that use the Filestore storage class before destroying the cluster.
+
+.Prerequisites
```

Comment:
If the user deleted the PVCs before destroying the cluster (as suggested in the paragraph above), there is no need to follow the instructions below. Should we remove them?

Reply:
Hmmm. Good point. Should we perhaps show how to delete PVC?

Reply:
Perhaps something like this would be OK?

```
List all PVCs that were created using the StorageClass `filestore-csi`:

$ oc get pvc --all-namespaces --field-selector=spec.storageClassName=filestore-csi

Delete all PVCs listed by the command above:

$ oc delete pvc-***
```

@chao007 since you are more familiar with this, could you confirm that the command above is correct (I don't have any cluster at hand right now)?

Reply:
Done.

Reply:
I checked this cmd but now work.
oc get pvc --all-namespaces --field-selector="spec.storageClassName=filestore-csi"
Error from server (BadRequest): Unable to find "/v1, Resource=persistentvolumeclaims" that match label selector "", field selector "spec.storageClassName=filestore-csi": "spec.storageClassName" is not a known field selector: only "metadata.name", "metadata.namespace"
My oc version
Client Version: 4.12.0-0.nightly-2022-12-04-160656

Reply:
`$ oc get pvc  -o json -A | jq -r '.items[] | select(.spec.storageClassName == "filestore-csi") .metadata.name'`
I appened -A to list all-namespaces.